Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ASM_Alert.yml

```diff
@@ -4015,28 +3063,75 @@ tasks:
     quietmode: 0
     isoversize: false
     isautoswitchedtoquietmode: false
-  "154":
-    id: "154"
-    taskid: c55df8d6-0c42-4c9f-82f4-2d272f1fe5cf
-    type: title
+  "169":
+    id: "169"
+    taskid: 57e40a4b-eeba-4618-8bac-298e0a56c081
+    type: regular
     task:
-      id: c55df8d6-0c42-4c9f-82f4-2d272f1fe5cf
+      id: 57e40a4b-eeba-4618-8bac-298e0a56c081
       version: -1
-      name: Prompt All Options - S3
-      type: title
+      name: Set playbook stage grid field (notification)
+      description: |-
+        Automation used to more easily populate a grid field.  This is necessary when you want to assign certain values as static or if you have context paths that you will assign to different values as well.  Instead of a value you can enter `TIMESTAMP` to get the current timestamp in ISO format.  Example of command:
```

Comment:
```suggestion
        Automation used to more easily populate a grid field.  This is necessary when you want to assign certain values as static or if you have context paths that you will assign to different values as well.  Instead of a value you can enter `TIMESTAMP` to get the current timestamp in ISO format.  Example of the command:
```